### DISCUSSION TITLE
关于Natasha的内存占用问题

### DISCUSSION THREAD

**Discussion by wosledon** [dotnetcore/Natasha#176] [General]
我使用Natasha构建了一个简单的例子，使用相同的Domain的情况下，模拟不断构建脚本，内存每次上升5~12M左右。大概设计场景是，前端编写脚本交由后端进行编译，我无法预期有哪些using是可以排除的🤔
![image](https://user-images.githubusercontent.com/49866401/230289830-d688a20b-73c1-4fcb-8de6-b708b0c358e0.png)

**Comment by NMSAzulX** [comment]
https://github.com/dotnetcore/Natasha/issues/210
